tests/test_extract.py

```diff
@@ -35,6 +35,16 @@ def test_extract(self):
                          np.array(['MultiPolygon']))
         self.assertTrue(len(gdf_mp)==4202)
         
+        gdf_mp2 = extract(OSM_FILE, 'multipolygons',  ['building', 'name', ])
```

Comment:
Is there a reason why all new tests have the value `name`? Is this needed?

Reply:
no, it works with any amount of keys. At least 1 is needed.

Reply:
Then a last question: is the empty element of the list intentional? I.e. the ', ' after the 'name' ?

Reply:
no, you can remove it if you want :)